solutions/gold/sumdiv.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: sumdiv
+source: Kilonova
+title: Sum of divisors
+author: Mihnea Brebenel
+---
+
+## Explanation
+
+The sum of all divisors of the integer $N$ can be efficiently computed in $\mathcal{O}(\sqrt{N})$ using the [divisor function](https://en.wikipedia.org/wiki/Divisor_function).
+The complexity comes from the prime factorization of $N$.
+
+The formula states that: Let $d_1^{p_1}, d_2^{p_2}, ..., d_k^{p_k}$ be the prime factorization of $N$, then the divisors' sum is $S = \prod_{i=1}^{k}\frac{d_i^{p_i+1} - 1}{d_i -1}$.
+
+## Implementation
+
```

Comment:
put a time complexity indicator here ik you said it above but it's for consistency

Reply:
Done